### PR TITLE
Study activity event enhancements

### PR DESCRIPTION
**Pull request by alxdarksage**

### CODE REVIEW

File: src/test/java/org/sagebionetworks/bridge/sdk/integration/StudyBurstTest.java [hunk 1]
```diff
@@ -131,6 +131,73 @@ private void setupSchedule(String originEventId, ActivityEventUpdateType burstUp
...
+
+        StudyActivityEventList list = usersApi.getStudyActivityEvents(STUDY_ID_1).execute().body();
+
+        StudyActivityEvent originEvent = findEventById(list, "custom:event1");
+        assertNull(originEvent.getAnswerValue());
+        assertNull(originEvent.getOriginEventId());
+        assertNull(originEvent.getStudyBurstId());
+        assertNull(originEvent.getPeriodFromOrigin());
+        assertEquals(Integer.valueOf(1), originEvent.getRecordCount());
+        assertEquals(timestamp1, originEvent.getTimestamp());
```
Comment (alxdarksage) [inline src/test/java/org/sagebionetworks/bridge/sdk/integration/StudyBurstTest.java:150]: Verify the client time zone comes back in the event and in the study burst events.

File: src/test/java/org/sagebionetworks/bridge/sdk/integration/Tests.java [hunk 2]
```diff
@@ -151,6 +153,26 @@ public static String randomIdentifier(Class<?> cls) {
...
+        for (T item : iterable) {
+            S retValue = func.apply(item);
+            if (value == null && retValue == null) {
+                return Optional.of(item);   
+            } else if (retValue != null && retValue.equals(value)) {
+                return Optional.of(item);
+            }
+        }
+        return Optional.empty();
+    }
```
Comment (alxdarksage) [inline src/test/java/org/sagebionetworks/bridge/sdk/integration/Tests.java:174]: Does Guava have an Iterables method?

Comment (alxdarksage) [reply src/test/java/org/sagebionetworks/bridge/sdk/integration/Tests.java:174]: Only with a predicate.

**Review by nategbrown9 — APPROVED** [review]
Looks good.

**Pull request merged**